docs/RELEASING.md

```
@@ -0,0 +1,133 @@
+## Version scheme
+
+| Channel   | Tag format                  | Example           |
+|-----------|-----------------------------|-------------------|
+| `beta`    | `vMAJOR.MINOR.(PATCH+1)-beta.N` | `v0.7.3-beta.1` |
+| `release` | `vMAJOR.MINOR.PATCH`        | `v0.7.3`          |
+
+### Why betas use `PATCH+1`
+
+The [semver spec §9](https://semver.org/#spec-item-9) gives pre-release versions *lower* precedence than the normal version they annotate:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)